packages/client/src/runtime/core/engines/data-proxy/DataProxyEngine.ts

```diff
@@ -158,10 +165,11 @@ export class DataProxyEngine extends Engine<DataProxyTxInfoPayload> {
 
     this.config = config
     this.env = { ...this.config.env, ...process.env }
-    this.inlineSchema = config.inlineSchema ?? ''
-    this.inlineDatasources = config.inlineDatasources ?? {}
-    this.inlineSchemaHash = config.inlineSchemaHash ?? ''
-    this.clientVersion = config.clientVersion ?? 'unknown'
+    this.inlineSchema = config.inlineSchema
+    this.inlineDatasources = config.inlineDatasources
+    this.inlineSchemaHash = config.inlineSchemaHash
+    this.clientVersion = config.clientVersion
```

Comment:
They used to be optional before, but not anymore (example: https://github.com/prisma/prisma/commit/49a66fa70742f70c9fed376473082485ad02d497#diff-84f89062eba89dc0d9ef5a8d43ee3ae512a780a4da288e57b6b937763748e2a9R66-R71)